feature-libs/cart/saved-cart/components/list/saved-cart-list.component.html

```diff
@@ -1,156 +1,154 @@
 <ng-container *ngIf="savedCarts$ | async as savedCarts">
-  <div class="container">
-    <div class="cx-saved-cart-list-header">
-      <h3>
-        {{
-          'savedCartList.savedCarts' | cxTranslate: { count: savedCarts.length }
-        }}
-      </h3>
-    </div>
+  <div class="cx-saved-cart-list-header">
```

Comment:
As mentioned in the DM,

there's something wrong with the listing page 

In epic branch, it's looking like this 
<img width="1138" alt="Screen Shot 2021-03-15 at 1 38 07 PM" src="https://user-images.githubusercontent.com/29099466/111197687-24dcbf00-8595-11eb-838b-8f75a34a717b.png">
<img width="600" alt="Screen Shot 2021-03-15 at 1 40 19 PM" src="https://user-images.githubusercontent.com/29099466/111197711-2b6b3680-8595-11eb-8e8b-2a608e81e431.png">

whereas, it is now looking like this

<img width="1207" alt="Screen Shot 2021-03-15 at 1 36 54 PM" src="https://user-images.githubusercontent.com/29099466/111197743-358d3500-8595-11eb-851c-53904f4a34e7.png">
<img width="720" alt="Screen Shot 2021-03-15 at 1 41 08 PM" src="https://user-images.githubusercontent.com/29099466/111197751-38882580-8595-11eb-8908-b7a682079507.png">